.devcontainer/.dev_config.yaml

```diff
@@ -305,6 +305,8 @@ service_name: metldata
 service_instance_id: 001
 kafka_servers: ["kafka:9092"]
 
+primary_artifact_name: embedded_public
+embedded_dataset_class_name: EmbeddedDataset
```

Comment:
Maybe call it primary_dataset_name so that it's clear that these belong together.

Reply:
Changed in b1142b1c72c3b538fb6e712b3ae4b583fe386e0f